rocksdb_replicator/replicated_db.cpp

```diff
@@ -134,13 +134,17 @@ RocksDBReplicator::ReplicatedDB::ReplicatedDB(
     folly::Executor* executor,
     const DBRole role,
     const folly::SocketAddress& upstream_addr,
-    common::ThriftClientPool<ReplicatorAsyncClient>* client_pool)
+    common::ThriftClientPool<ReplicatorAsyncClient>* client_pool,
+    const std::string& replicator_zk_cluster,
+    const std::string& replicator_helix_cluster)
```

Comment:
Should we create another constructor with these two new arguments? That would prevent any changes to existing callers. WDYT?

Reply:
They have default values:
```
                   const std::string& replicator_zk_cluster = "",
                   const std::string& replicator_helix_cluster = "");
```
(see https://github.com/pinterest/rocksplicator/pull/544/files/9579632255eea6b3b67e4edd8eddb781780d0cc8#diff-cb14f07e0183bff583746dbb5adf040de197c3cd1dfcb08369770004f083955bR187-R188) 
so existing callers should not be affected